scala/scala.bzl

```diff
@@ -34,25 +34,38 @@ def _get_all_runfiles(targets):
       runfiles += _get_runfiles(target)
     return runfiles
 
-def _adjust_resources_path(path):
+
+def _adjust_resources_path_by_strip_prefix(path,resource_strip_prefix):
+    if not path.startswith(resource_strip_prefix):
+      fail("Resource file %s is not under the specified prefix to strip" % path)
+
+    clean_path=path[len(resource_strip_prefix):]
+    return resource_strip_prefix, clean_path
+
+def _adjust_resources_path_by_general_rules(path):
```

Comment:
general_rules meaning the default rules? WDYT about _adjust_resources_path_by_default_prefixes?